doc/specs/#885 - winrt Terminal Settings.md

```diff
@@ -2,7 +2,7 @@
 author: Carlos Zamora @carlos-zamora
 created on: 2020-07-10
 last updated: 2020-07-10
-issue id: [#885]
+issue id: /#885

```

Comment:
`/`?

Reply:
I need to figure out how to make it appear properly hahaha

Reply:
either do what our other specs do, or just don't?

or use `[#885](https://actual/link/to/issues/885)`?